packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -1488,6 +1345,331 @@ class _DialState extends State<_Dial> with SingleTickerProviderStateMixin {
   }
 }
 
+class _TimePickerInput extends StatefulWidget {
+  const _TimePickerInput({
+    Key key,
+    @required this.initialSelectedTime,
+    @required this.helpText,
+    @required this.onChanged,
+  }) : assert(initialSelectedTime != null),
+        super(key: key);
+
+  /// The time initially selected when the dialog is shown.
+  final TimeOfDay initialSelectedTime;
+
+  /// Optionally provide your own help text to the time picker.
+  final String helpText;
+
+  final ValueChanged<TimeOfDay> onChanged;
+
+  @override
+  __TimePickerInputState createState() => __TimePickerInputState();
+}
+
+class __TimePickerInputState extends State<_TimePickerInput> {
```

Comment:
I think you accidentally put one extra `_` in the widget here.

Reply:
You're right, I blame the refactor tool haha